libs/s25main/network/GameClient.cpp

```diff
@@ -54,6 +54,23 @@
 #include <helpers/chronoIO.h>
 #include <memory>
 
+namespace {
+void copyFileIfPathDifferent(const boost::filesystem::path& src_path, const boost::filesystem::path& dst_path)
```

Comment:
we should put this function somewhere like libutil or so

Reply:
Can we do this in another merge request or is there a location where i can move the function without having to update a submodule. If i understand the submodules correct i have to make a PR for the submodule repo and then after this is merged back i have to change the pointer to the submodule in a PR of the s5client repo.

Reply:
Yes, IMO we can do that after this PR